packages/components/src/components/Log/Log.scss

```diff
@@ -53,6 +53,20 @@ pre.tkn--log {
     right: 0;
   }
 
+  .bx--copy-btn.scroll-btn {
```

Comment:
I'm not seeing this style applied, instead the buttons just appear at the bottom of the log.